src/build/supportedExtensions.js

```diff
@@ -96,9 +96,9 @@ exports.extensions = {
     { icon: 'image', extensions: ['jpeg', 'jpg', 'gif', 'png', 'bmp', 'ico'] },
     { icon: 'ionic', extensions: ['ionic.project'], filename: true },
     { icon: 'ionic', extensions: ['ionic.config.json'], filename: true },
-    { icon: 'jade', extensions: ['jade', 'pug', 'tag'] },
```

Comment:
Cool. Yes, I only see a `Riot extension`. Any language extension for Jade/Pug... so let's removed as you originally intended. 👍